setup.py

```diff
@@ -117,8 +117,14 @@
 if platform.system() == "Darwin":
```

Comment:
Hi psobot, how are you? I'm ashamed and I could be wrong but,

What you think about decouple the arrays append in function by the OS?